source/manual/deploying.html.md

```diff
@@ -17,27 +17,48 @@ If you are responding to a security incident, follow the steps to [deploy fixes
 [reg]: https://gds.blog.gov.uk/2012/11/02/regular-releases-reduce-risk
 [sec]: /manual/deploy-fixes-for-a-security-vulnerability.html
 
-## 1. Test on integration
+## Deploying a branch
+
+It can be useful to test your branch in a non-local environment before it is merged to master. Note that you should only do this on Integration, unless you have a good reason to deploy to a different environment.
+
+As you'll be deploying to Integration, note that your deployment could get overwritten at any time as apps on Integration are automatically overwritten by their master branches when PRs are merged.
+
+Before deploying anything, check whether there is already a non-master branch deployed by looking at the Release application on production. (The Release app on staging and integration won't tell you, as they're synced from production overnight and aren't otherwise updated). If a non-master branch has been deployed, try and ask the person who deployed it if they're finished.
```

Comment:
If you don't yet have production access, another way to find this out is to [look at the build history in Jenkins](https://deploy.integration.publishing.service.gov.uk/job/Deploy_App/).

Reply:
Thanks, I'll add that in 👍